### ISSUE TITLE
Add website code to a branch on this repo?

### ISSUE THREAD

**Issue by djinnome** [pnnl-predictive-phenomics/clean-contact#1]
Hi @yuxin212 , can you add the website code at [https://ersa.guans.cs.kent.edu/](https://ersa.guans.cs.kent.edu/) to this repository?

**Comment by yuxin212** [comment]
Yes. The website code is under `website` branch. There is a very brief `README` file for the website code. Please read through the `README` under `clean-contact-website-backend` folder. Please email me if there is any issue with it.
